sql/core/src/main/scala/org/apache/spark/sql/json/JSONRelation.scala

```diff
@@ -28,10 +32,10 @@ private[sql] class DefaultSource extends RelationProvider with SchemaRelationPro
   override def createRelation(
       sqlContext: SQLContext,
       parameters: Map[String, String]): BaseRelation = {
-    val fileName = parameters.getOrElse("path", sys.error("Option 'path' not specified"))
+    val path = parameters.getOrElse("path", sys.error("Option 'path' not specified"))
```

Comment:
I think we should define common data source options like `path` in a single place. But we can leave it to another PR.